thunder/core/dtypes.py

```diff
@@ -144,6 +144,8 @@ def __init__(self, name, shortname, *, bytes, is_weak):
 
 uint8 = unsignedinteger("uint", "ui", bytes=1, is_weak=False)
 uint8_ = unsignedinteger("uint", "ui", bytes=1, is_weak=True)
+uint64 = unsignedinteger("uint", "ui", bytes=8, is_weak=False)
+uint64_ = unsignedinteger("uint", "ui", bytes=8, is_weak=True)
```

Comment:
I am not exposing/including `uint64` anywhere else since it otherwise gets picked by tests causing errors.
I only need `uint64` to correctly declare the `philox_seed` and `philox_offset` for torch 2.7.0 and above.